configure.ac

```diff
@@ -563,7 +563,7 @@ PHP_CHECK_CPU_SUPPORTS([avx])
 PHP_CHECK_CPU_SUPPORTS([avx2])
 
 dnl Check for members of the stat structure
-AC_STRUCT_ST_BLKSIZE
+AC_CHECK_MEMBERS([struct stat.st_blksize, struct stat.st_rdev])
 dnl AC_STRUCT_ST_BLOCKS will screw QNX because fileblocks.o does not exists
```

Comment:
Is this still relevant? I'm asking because the comment refers to `AC_STRUCT_ST_BLOCKS` which would be confusing if it will be replaced in the line above.

Reply:
@cmb69 The `AC_STRUCT_ST_BLOCKS` is used in the line below. Note, that the macro being replaced in this patch is `AC_STRUCT_ST_BLKSIZE` which is not related to that comment.

I think the comment and the adjustments for QNX should stay for now, but I'll check it out what's up with this also. It's from the times of PHP 4 or pre era and is very old and most likely not relevant adjustments anymore also, yes.

Reply:
> The AC_STRUCT_ST_BLOCKS is used in the line below.

Oops! Feel free to ignore the comment, then. :)